pkg/query-service/contextlinks/links.go

```diff
@@ -146,7 +147,7 @@ func PrepareLinksToLogs(start, end time.Time, filterItems []v3.FilterItem) strin
 // by clause, in which case we replace it with the actual value for the notification
 // i.e Severity text = WARN
 // If the Severity text is not part of the group by clause, then we add it as it is
-func PrepareFilters(labels map[string]string, whereClauseItems []v3.FilterItem, groupByItems []v3.AttributeKey, keys map[string]v3.AttributeKey) []v3.FilterItem {
+func PrepareFilters(alertTypeLogs bool, labels map[string]string, whereClauseItems []v3.FilterItem, groupByItems []v3.AttributeKey, keys map[string]v3.AttributeKey) []v3.FilterItem {
```

Comment:
Please remove the logs specific logic from here. The intent of this is to be generic. Treat the `labels` as if they are user given filters and enrich them appropriately.